beacon-chain/core/helpers/beacon_committee.go

```diff
@@ -58,6 +59,29 @@ func SlotCommitteeCount(activeValidatorCount uint64) uint64 {
 	return committeesPerSlot
 }
 
+// AttestationCommittees returns beacon state committees that reflect attestation's committee indices.
+func AttestationCommittees(ctx context.Context, st state.ReadOnlyBeaconState, att ethpb.Att) ([][]primitives.ValidatorIndex, error) {
+	var committees [][]primitives.ValidatorIndex
+	if att.Version() >= version.Electra {
```

Comment:
I find it odd that we could pass in a phase 0 state but an Electra attestation, and this will work as expected. Maybe it's okay by design.

Reply:
That's a valid point. I asked myself the same thing but there are many other places in the codebase where we don't check if objects' versions match e.g. `ExecuteStateTransition`. Best thing to do would be to come up with a design pattern for these kind of checks and enforce them everywhere in a single PR.